client/cody-ui/src/chat/inputContext/ChatInputContext.module.css

```diff
@@ -50,15 +50,4 @@
 .index-present,
 .index-missing {
     font-weight: bold;
-}
-
-.indexStatusOnHover,
-a:hover .indexStatus {
-    display: none;
-}
-
-.indexStatus,
-a:hover .indexStatusOnHover {
-    display: inline;
-    color: var(--vscode-list-warningForeground, #fdf1cc);
-}
+}
```

Comment:
```suggestion
.indexStatus {
    display: inline;
    color: var(--vscode-list-warningForeground, #fdf1cc);
}
```
Should we keep the `color: var(--vscode-list-warningForeground, #fdf1cc);`? Removing this caused the status text to fallback using color for links, which is green on my end 👀 :
<img width="521" alt="image" src="https://github.com/sourcegraph/sourcegraph/assets/68532117/2dd87b1b-eb5d-4eca-ae6f-b4561d1976fd">